app/models/site.rb

```diff
@@ -42,6 +43,16 @@ def needs_geolocation?
     )
   end
 
+  def add_travel_to_work_area_and_london_borough
+    TravelToWorkAreaAndLondonBoroughJob.perform_later("Site", id) if needs_travel_to_work_area_and_london_borough_updated?
```

Comment:
On the back of https://github.com/DFE-Digital/teacher-training-api/pull/1756/files#r572778403 this could be simplified by passing in the site id only ...